demos/projects/ESPRESSIF/adu/config/demo_config.h

```diff
@@ -218,10 +218,15 @@
  */
 #define democonfigIOTHUB_PORT            8883
 
+/**
+ * @brief Defines device restart function, used by the common sample modules.
+ */
+#define configRestartDevice()    esp_restart()
+
 /**
  * @brief Defines configRAND32, used by the common sample modules.
  */
-#define configRAND32()    ( rand() / RAND_MAX )
+#define configRAND32()           ( rand() / RAND_MAX )
```

Comment:
```suggestion
#define democonfigRAND32()           ( rand() / RAND_MAX )
```

Reply:
It looks like some of the samples define this in demo_config and some define it in FreeRTOSConfig, so I think that's why the prefix is just `config`. Probably best to leave it as is for now then?
https://github.com/Azure-Samples/iot-middleware-freertos-samples/search?q=configRAND32